src/treeChooser/treeChooser.html

```diff
@@ -2,15 +2,15 @@
   <div class="treeChooser-input"
        ng-class="{'treeChooser-input-focused': vm.focused}">
     <span ng-if="vm.enablePills"
-          ng-click="vm.removeFromModel(item); focusInput()"
+          ng-click="vm.removeFromModel(item); focusInput(); vm.show();"
```

Comment:
Verify if vm.show() is still needed since vm.show() will now be called on ng-focus